Mapping/Loader/ReflectionLoader.php

```diff
@@ -0,0 +1,221 @@
+<?php
+
+/*
+ * This file is part of the DunglasApiBundle package.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Dunglas\ApiBundle\Mapping\Loader;
+
+use Dunglas\ApiBundle\Mapping\Factory\AttributeMetadataFactoryInterface;
+use Dunglas\ApiBundle\Mapping\AttributeMetadataInterface;
+use Dunglas\ApiBundle\Mapping\ClassMetadataInterface;
+
+/**
+ * Uses serialization groups or alternatively reflection to populate attributes.
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class ReflectionLoader implements LoaderInterface
+{
+    const DEFAULT_IDENTIFIER = 'id';
```

Comment:
Can be passed in constructor instead, to expose this configuration in the container, and so possibly in the bundle's extension.